src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -42,7 +42,7 @@ $font-stack: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial"
   font-family: "Source Sans Pro";
   font-style: normal;
   font-weight: 300;
-  src: url(/generated/sourcesanspro-light-webfont.eot?#iefix) format("embedded-opentype"),
+  src: url("/generated/sourcesanspro-light-webfont.eot?#iefix") format("embedded-opentype"),
```

Comment:
Can we revert the changes in here?

Reply:
Hey @jbalboni! Sure thing, I'll give it a try. For some reason, the linter kept screaming about it when I was trying to commit.